### PR TITLE
🐛 Fixing currency issue

### PR DESCRIPTION
**Pull request by xoscar**
Fixes https://github.com/open-telemetry/opentelemetry-demo/issues/506

## Changes

Fixes a bug where the prices weren't changing anywhere that wasn't the home page.

### CODE REVIEW

File: src/frontend/services/ProductCatalog.service.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ const ProductCatalogService = () => ({
 
     return {
       ...product,
-      price: await this.getProductPrice(product.priceUsd!, currencyCode),
+      priceUsd: await this.getProductPrice(product.priceUsd!, currencyCode),
```
Comment (xoscar) [inline src/frontend/services/ProductCatalog.service.ts:32]: Wasn't replacing the right entry 😢

**Review by austinlparker — DISMISSED** [review]

**Review by julianocosta89 — DISMISSED** [review]
Thanks for taking a look at this @xoscar 🤩 

We are still missing the same changes on the frontend cart page.
image

**Comment by xoscar** [comment]
Hey @julianocosta89 I just pushed a fix, do you mind taking a look? Thanks!

**Comment by cartersocha** [comment]
No merge until additional review

**Review by julianocosta89 — APPROVED** [review]
Awesome!
Thx again for taking care of this @xoscar!

**Comment by austinlparker** [comment]
Merged as #522

**Pull request closed without merging**